lua/null-ls/utils/init.lua

```diff
@@ -211,6 +211,20 @@ M.handle_function_opt = function(opt, ...)
     return vim.deepcopy(opt)
 end
 
+--- gets git root
+---@return string
+M.get_git_root = function()
+    local result = vim.system({
+        "git",
```

Comment:
consider explicitly set search directory with `git -C ${M.get_root()}` (and likewise for other VCS) or set cwd on launch.

Reply:
done and tested on local monorepo project. both local bin and monorepo root bin can be found.